shelf_num: 2

shelf0:
  pos: [0.97, 1, 0]
  quat: [0, 0, -0.707, 0.707]
  board_height_upper: 0.95
  board_height_lower: 0.66

shelf1:
  pos: [0, 1, 0]
  quat: [0, 0, -0.707, 0.707]
  board_height_upper: 0.95
  board_height_lower: 0.66
